src/features/Courses/CoursesDetailPage/index.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { Container, Pagination } from '@edx/paragon';
+
+import { fetchClassesData } from 'features/Classes/data/thunks';
+import { fetchCoursesData } from 'features/Courses/data/thunks';
+import { updateCurrentPage } from 'features/Courses/data/slice';
+import CourseDetailTable from 'features/Courses/CourseDetailTable';
+import { fetchClassesDataSuccess } from 'features/Classes/data/slice';
+import { initialPage } from 'features/constants';
+
+import 'features/Courses/CoursesDetailPage/index.scss';
+
+const CoursesDetailPage = () => {
+  const dispatch = useDispatch();
+  const { classId } = useParams();
+
+  const [currentPage, setCurrentPage] = useState(initialPage);
+
+  const defaultCourseInfo = {
+    numberOfStudents: '-',
+    numberOfPendingStudents: '-',
+  };
+
+  const courseInfo = useSelector((state) => state.courses.table.data)
+    .find((course) => course?.masterCourseName === classId) || defaultCourseInfo;
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const classes = useSelector((state) => state.classes.table);
+
+  const handlePagination = (targetPage) => {
+    setCurrentPage(targetPage);
+    dispatch(updateCurrentPage(targetPage));
+  };
+
+  useEffect(() => {
+    if (institution.id) {
+      dispatch(fetchClassesData(institution.id, 1, classId));
+      dispatch(fetchCoursesData(institution.id, 1, null));
+    }
+
+    const initialState = {
+      results: [],
+      count: 0,
+      numPages: 0,
+    };
+
+    return () => dispatch(fetchClassesDataSuccess(initialState));
+  }, [dispatch, institution.id, classId]);
+
+  return (
+    <Container size="xl" className="px-4">
+      <div className="d-flex justify-content-between mb-3 flex-column flex-sm-row">
+        <div className="d-flex align-items-center mb-3">
+          <Link to="/courses" className="mr-3 link">
+            <i className="fa-solid fa-arrow-left" />
+          </Link>
+          <h3 className="h2 mb-0">{classId}</h3>
+        </div>
+
+        <div className="card-container d-flex justify-content-around align-items-center">
+          <div className="d-flex flex-column align-items-center">
+            <p className="title">Students enrolled</p>
+            <span className="value">
+              {courseInfo.numberOfStudents}
+              {' / '}
+              {courseInfo.numberOfStudents + courseInfo.numberOfPendingStudents}
```

Comment:
```suggestion
              {totalStudents}
```
What do you think about assign this operation into a variable to make the code more readable? @01001110J